app/views/users/registrations/new.html.erb

```diff
@@ -0,0 +1,40 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "users/shared/error_messages", resource: resource %>
+
+ <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+ </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+  
+  
+    <div class="field">
+    <%= f.label :post_count %><br />
+    <%= f.number_field :post_count, autofocus: true, autocomplete: "post_count", value: 0%>
+  </div>
+
```

Comment:
- [ ] [OPTIONAL]
It will be a good idea if we remove the post_count input field from the signup form, as it doesn't form a requirement for signup.

**Your signup form:**
![image](https://github.com/Rachelwebdev/blog-app/assets/12745474/700aa610-daa5-4e1c-827c-c4556af5d9a7)